app/src/main/res/layout/fragment_form_customer_address.xml

```diff
@@ -63,7 +63,8 @@
                             android:id="@+id/et_city"
                             android:inputType="text"
                             android:layout_height="wrap_content"
-                            android:layout_width="match_parent"/>
+                            android:layout_width="match_parent"
+                            android:digits="abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"/>
```

Comment:
Don't use hardcoded string. Add different digits patterns in strings.xml and use them.